lib/Stampie/MailerInterface.php

```diff
@@ -12,12 +12,12 @@
 interface MailerInterface
 {
     /**
-     * @param AdapterInterface $adapter
+     * @param HttpClient $adapter
      */
-    function setAdapter(AdapterInterface $adapter);
+    function setAdapter(HttpClient $adapter);
 
     /**
-     * @return AdapterInterface
+     * @return HttpClient
      */
     function getAdapter();
```

Comment:
I would also remove the setters for the server token btw, making things more flexible for each implementation about how authentication is handled.